diff --git a/node_modules/protocol-buffers/compile.js b/node_modules/protocol-buffers/compile.js
index 1cdad22..67a1cb2 100644
--- a/node_modules/protocol-buffers/compile.js
+++ b/node_modules/protocol-buffers/compile.js
@@ -32,6 +32,8 @@ const defaultValue = function (f, def) {
   if (f.map) return '{}'
   if (f.repeated) return '[]'
 
+  return 'undefined';
+
   switch (f.type) {
     case 'string':
       return isString(def) ? def : '""'
